modules/benchmarks/e2e_test/largetable_perf.ts

```diff
@@ -65,10 +65,10 @@ describe('largetable benchmark perf', () => {
         }).then(done, done.fail);
       });
 
-      it('should run for iv', (done) => {
+      it('should run for ivy', (done) => {
         runTableBenchmark({
-          id: `largeTable.iv.${worker.id}`,
-          url: 'all/benchmarks/src/largetable/iv/index.html',
+          id: `largeTable.ivy.${worker.id}`,
+          url: 'all/benchmarks/src/largetable/ivy/index.html',
```

Comment:
The path's should say `render3` not `ivy`. I would also not include `ivy` anywhere in the source as it is a code name. Just say render3.

Reply:
This benchmark is actually `iv`, the original prototype from Bertrand.

Reply:
good, point, we should keep it as `iv` than